README.md

```diff
@@ -531,3 +531,47 @@ console via the debug UART adapter / the USB console provided by barebox
 and execute `wd -x` on the shell. E.g.:
 
     barebox@Linux Automation Test Automation Controller (TAC):/ wd -x
+
+Contributing
+------------
+
+Thank you for thinking about contributing to this project!
+Changes should be submitted via a
+`Github pull request <https://github.com/linux-automation/meta-lxatac/pulls>`_.
```

Comment:
This looks like ReStructuredText syntax but the README is in Markdown.

Rendered result:

![dco](https://github.com/linux-automation/meta-lxatac/assets/1273320/64568ce1-7864-4cff-be6b-433ba72585a4)